src/actions/Bond.actions.js

```diff
@@ -276,6 +269,20 @@ export const redeemBond =
         fetchPendingTxns({ txnHash: approveTx.hash, text: "Redeeming " + bondName(bond), type: pendingTxnType }),
       );
       await redeemTx.wait();
+      await dispatch(calculateUserBondDetails({ address, bond, networkID, provider }));
+
+      const ohmContract = new ethers.Contract(addresses[networkID].OHM_ADDRESS, ierc20Abi, provider);
+      const ohmBalance = await ohmContract.balanceOf(address);
+      const sohmContract = new ethers.Contract(addresses[networkID].SOHM_ADDRESS, ierc20Abi, provider);
+      const sohmBalance = await sohmContract.balanceOf(address);
+      await dispatch(
+        fetchStakeSuccess({
```

Comment:
Should this actually be a separate action? it feels weird to call "stake_success"  from the bond actions. and infact thats not even whats happening.

Reply:
Yeah I think a better name for that would be like `fetchBalances`. But I put this in here because after the user redeems their bonds, I believe their ohm/sohm balance will change. Depending on if they just redeem or redeem and autostake.